### DISCUSSION TITLE
Studebt Account Verifications

### DISCUSSION THREAD

**Discussion by ahrufcodes** [community/community#49152] [GitHub Education]
Hi,

My application has been rejected twice for the Github student package, I guess because my school is remote hence my current location is different from my school's location. I also find it a bit difficult to capture my documents because my webcam is pretty bad.
Can I send my documents to a specific email address? How best can I resolve this?

Thank You

**Comment by maahaMaya** [comment]
Some point to remember that. 
1. email address is provided by the school's or university i.e.  ahrufcodes@school.edu  [ .edu extension in email address]
2. use external web camera.

**Reply by StephenAnimates** [reply]
> Some point to remember that. 
> 
> 1. email address is provided by the school's or university i.e.  ahrufcodes@school.edu  [ .edu extension in email address]
> 
> 2. use external web camera.

Why? Why should I be required to use a webcam? Why is this the only option?

**Comment by ahrufcodes** [comment]
Hi,

Yes I have the email address provided by my school and I have linked it
with my GitHub.

How best do you suggest I use an external webcam, because I don't see such
option on the page.

Best regards

**Reply by maahaMaya** [reply]
First buy external web cam for computers or laptop and connect to the laptop and use it.

**Reply by maahaMaya** [reply]
Hey can u please mark as answered

**Comment by 1svyatt** [comment]
Q
